src/telegrafs/actions/thunks.ts

```diff
@@ -76,17 +85,17 @@ export const getTelegrafs = () => async (
 }
 
 export const createTelegraf = (telegraf: Telegraf) => async (
-  dispatch: Dispatch<Action>,
-  getState: GetState
+  dispatch: Dispatch<Action>
 ) => {
   try {
-    const state = getState()
-    const labels = getLabels(state, telegraf.labels)
```

Comment:
A new telegraf config doesn't expect to have any labels, so no need to call `getLabels`.